src/main/java/uk/gov/register/RegisterApplication.java

```diff
@@ -99,7 +99,9 @@ public void run(RegisterConfiguration configuration, Environment environment) th
         DBI jdbi = dbiFactory.build(environment, configuration.getDatabase(), "postgres");
 
         Flyway flyway = configuration.getFlywayFactory().build(configuration.getDatabase().build(environment.metrics(), "flyway_db"));
-
+        flyway.setBaselineVersionAsString("0");
+        flyway.migrate();
```

Comment:
Can we not do this please? The whole point of #95 is to remove this call to `flyway.migrate()`